src/pages/saksbehandling/steg/faktablokk/faktablokker/UtenlandsOppholdFaktablokk.tsx

```diff
@@ -0,0 +1,76 @@
+import * as DateFns from 'date-fns';
+import { Element, Normaltekst } from 'nav-frontend-typografi';
+import React from 'react';
+import { IntlShape } from 'react-intl';
+
+import { kalkulerTotaltAntallDagerIUtlandet, Utlandsdatoer } from '~lib/dateUtils';
+import { useI18n } from '~lib/hooks';
+
+import Faktablokk from '../Faktablokk';
+
+import messages from './faktablokker-nb';
+import styles from './faktablokker.module.less';
+import { FaktablokkProps } from './faktablokkUtils';
+
+const UtenlandsOppholdFaktablokk = (props: FaktablokkProps) => {
+    const intl = useI18n({ messages });
+
+    return (
+        <Faktablokk
+            tittel={intl.formatMessage({ id: 'display.fraSøknad' })}
+            tittelType={props.tittelType}
+            fakta={[
+                {
+                    tittel: intl.formatMessage({ id: 'utenlandsOpphold.antallDagerSiste90' }),
+                    verdi: kalkulerTotaltAntallDagerIUtlandet(
+                        props.søknadInnhold.utenlandsopphold.registrertePerioder
+                    ).toString(),
+                },
+                {
+                    tittel: intl.formatMessage({ id: 'utenlandsOpphold.antallDagerPlanlagt' }),
+                    verdi: kalkulerTotaltAntallDagerIUtlandet(
+                        props.søknadInnhold.utenlandsopphold.planlagtePerioder
+                    ).toString(),
+                },
+                {
+                    tittel: intl.formatMessage({ id: 'utenlandsOpphold.datoerSiste90' }),
+                    verdi: visDatoer(props.søknadInnhold.utenlandsopphold.registrertePerioder, intl),
+                },
+                {
+                    tittel: intl.formatMessage({ id: 'utenlandsOpphold.datoerPlanlagt' }),
+                    verdi: visDatoer(props.søknadInnhold.utenlandsopphold.planlagtePerioder, intl),
+                },
+            ]}
+        />
+    );
+};
+
+const visDatoer = (datesArray: Utlandsdatoer, intl: IntlShape) => {
+    if (!datesArray || datesArray?.length === 0) return intl.formatMessage({ id: 'fraSøknad.ikkeRegistert' });
```

Comment:
`!datesArray` kan väl aldrig ske?

Reply:
Utlandsdatoer er en nullable type. sam må samtidig sjekke for lengden for å få ut tekst-stringen

Reply:
åja... tänker att det är litt rart att typen är innbyggt nullable. mer naturligt att isåfall det hade stått `Nullable<Utlandsdatoer>`.

Reply:
Har endret på det uansett nå.